Automattic/wp-calypso
# Stats: Polish post detail tables with existing stats table styles

#### Proposed Changes

* Polish and reuse the existing stats table and control styles on the table section of the post detail page.
* Gate the featured changes on `PostMonths` and `PostWeeks` by the flag `stats/enhance-post-detail`.
* Extract the shared stats table styles as a separate file.

#### Testing Instructions



> For a11n, if you have no idea about the testing data on the tables, please refer to `p1671729060450249/1671728562.298969-slack-C0438NHCLSY`.

* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Post Detail` page (`/stats/post/${post-id}/${site-id}`).
* Ensure the post detail page works as previously.
* Append the feature flag `?flags=stats/enhance-post-detail` to the URL.
* Ensure the post detail page table styles are polished in line with the design.

##### Months and years table

| Before | After |
| --- | --- |
| <img width="1100" alt="total_before" src="https://user-images.githubusercontent.com/6869813/209185381-c4fa5690-2f8c-46d9-bb63-7e048220ef66.png"> | <img width="1087" alt="total_after_3" src="https://user-images.githubusercontent.com/6869813/209195493-cfc62951-4435-4aa1-9d9b-26fd7833bc95.png"> |

##### Average per day table

| Before | After |
| --- | --- |
| <img width="1096" alt="average_before" src="https://user-images.githubusercontent.com/6869813/209185629-0f64150a-66f5-4106-b939-8b9c927cb1ba.png"> | <img width="1094" alt="average_after_3" src="https://user-images.githubusercontent.com/6869813/209195558-d94960ae-0e44-4b5a-9426-2752bd4f3478.png"> |

##### Recent weeks table

| Before | After |
| --- | --- |
| <img width="1104" alt="week_before" src="https://user-images.githubusercontent.com/6869813/209193372-224d1cd9-2f08-4bd2-bb5c-bfead6b56f74.png"> | <img width="1091" alt="week_after_3" src="https://user-images.githubusercontent.com/6869813/209195603-ced4f604-7e73-48a7-b1fa-62ea3ff36689.png"> |

#### Pre-merge Checklist





Related to #70671
